### ISSUE TITLE
Unknown characters in resulting pdf

### ISSUE THREAD

**Issue by rastkojokic** [pdffillerjs/pdffiller#58]
Hey guys, trying to do a proof of concept filling out pdf form.

This is my code:

```
var pdfFiller   = require('pdffiller');
var sourcePDF = "tmp/pdf-forms/form-1.pdf";
var destinationPDF =  "tmp/created-pdf-forms/filled-form-1.pdf";

var data = {
    "q19_title19": "Some title",
    "q27_company27": "Foo Bar Company"
};

console.log('Start filling');
pdfFiller.fillForm(sourcePDF, destinationPDF, data, function(err) {
//pdfFiller.generateFDFTemplate(sourcePDF, null, function(err, data) {

    console.log(data);
    if (err) {
        console.log(err);
        throw err;
    }

    console.log("Form filled successfully");
});
```

And the resulting document is this:

[filled-form-1.pdf](https://github.com/pdffillerjs/pdffiller/files/885309/filled-form-1.pdf)

**Comment by maximilianschmidt** [comment]
Hi there, 

Maybe try changing the font of the form fields (in adobe right click form field -> settings -> appearance). That helped me when i encountered a problem with the rendered text. 

However i am encountering a (maybe) similar problem:

When filling a pdf with strings containing special characters, some are displayed in the rendered form and some are not. for example, the string 
`"Wypełnić Pełnomocnictwo, podpisać i wstawić pieczątkę." `
renders to 
`"Wypełni Pełnomocnictwo, podpisa i wstawi piecz tk ." `

The "ć, ą and ę" are missing and are rendered as a blank or as a square, but the "ł" is rendered correctly. 

Now the funny thing is, when i open the rendered pdf in adobe, edit the font of the formfield under Settings -> Appearance to Helvetica or Times or whatever, it doesnt really matter, the characters are all rendered correctly (`"ć, ą and ę"` are showing up in the form field). 

Even if i already set the font of the field to Helvetica (characters dont show up), save it, render the document, switch it to Times (characters do show up) and then back to Helvetica (characters also do show up). 

Anyone got an idea?

**Comment by satyajitnayk** [comment]
Yes, by changing font of a fillable field to a commonly available font such as `Helvatica` it works just fine.
<img width="390" alt="image" src="https://github.com/pdffillerjs/pdffiller/assets/32722867/72a33181-8325-4628-b9b1-d2e841fcae83">
